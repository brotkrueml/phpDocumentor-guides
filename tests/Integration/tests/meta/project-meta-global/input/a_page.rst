======
A Page
======

Lorem Ipsum Dolor.
