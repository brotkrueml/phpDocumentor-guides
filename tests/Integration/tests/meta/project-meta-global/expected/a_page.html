<!DOCTYPE html>
<html lang="en">
<head>
    <title>A Page - My Project</title>

</head>
<body>
    <div class="section" id="a-page">
    <h1>A Page</h1>

            <p>Lorem Ipsum Dolor.</p>
    </div>

</body>
</html>
